_sass/hub.scss

```diff
@@ -103,6 +103,26 @@
   }
 }
 
+.hub-feedback-button {
+  border: 2px solid #e2e2e2;
+  color: #A0A0A1;
+  padding-left: 0;
+  padding-right: 5rem;
+  font-size: 1rem;
+  width: 13rem;
+  :hover {
```

Comment:
I think these `:hover` styles might be unnecessary. If you remove them the button text still turns black on hover since the button is using the `btn` class.